packages/x-charts/src/BarChart/types.ts

```diff
@@ -18,12 +18,23 @@ export interface ProcessedBarSeriesData {
   yOrigin: number;
 }
 
+export const BorderRadiusSide = {
+  top: 'top',
+  bottom: 'bottom',
+  left: 'left',
+  right: 'right',
+} as const;
+
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export type BorderRadiusSide = (typeof BorderRadiusSide)[keyof typeof BorderRadiusSide];
```

Comment:
This is a bit confusing, the BorderRadiusSide has two different meanings, one if it is used as value and one if it is used as a type.